source/manual/github-unavailable.html.md

```diff
@@ -62,7 +62,7 @@ docker push $REGISTRY/$REPO:$IMAGE_TAG
 1. Find the `image:` field for the `app` container. It should look something like `172025368201.dkr.ecr.eu-west-1.amazonaws.com/<app-name>:release-2e902e3df274a00bbabba7ccf84cbef96ccc9b9e`.
 1. Update the tag part of the `image:` value to the new image tag that you pushed to ECR. The part you are changing should look something like `release-2e902e3df274a00bbabba7ccf84cbef96ccc9b9e`.
 1. Click `Save`. Argo CD will start the deployment, which should complete in under ten minutes.
-1. After confirming deployment of the app re-enable auto-sync for the `app-config` application:
+1. If you are following the 2ndline drill or when Github is available, after confirming deployment of the app re-enable auto-sync for the `app-config` application:
```

Comment:
I think it would be better to do the cleanup in a separate checklist:

```suggestion

When GitHub is available again (or when you've completed the above as part of a Technical 2nd Line drill), return things to normal by re-enabling the auto-sync:

```